unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/FriendsController.cs

```diff
@@ -15,14 +16,22 @@ public class FriendsController : IFriendsController
 
         private readonly IFriendsApiBridge apiBridge;
         private readonly ISocialApiBridge socialApiBridge;
+
+        // Used only when social client feature flag is false
         private readonly Dictionary<string, FriendRequest> friendRequests = new ();
+        private readonly Dictionary<string, FriendRequest> incomingFriendRequestsById = new ();
+        private readonly Dictionary<string, FriendRequest> outgoingFriendRequestsById = new ();
+        private readonly SortedList<long, FriendRequest> incomingFriendRequestsByTimestamp = new ();
```

Comment:
the most recent friend requests should be placed first in the list. To be able to do so you should initialize the list like this:
```
private class FirstAreMostRecent : IComparer<long>
{
    public int Compare(long x, long y) =>
    y.CompareTo(x);
}

private readonly SortedList<long, FriendRequest> incomingFriendRequestsByTimestamp = new (new FirstAreMostRecent());
```